app/mailers/report_mailer.rb

```diff
@@ -1,30 +1,32 @@
 class ReportMailer < ActionMailer::Base
-  default :from => AppConfig.mail.sender_address
+  default from: AppConfig.mail.sender_address
 
-  def new_report(type, id)
+  def self.new_report(type, id)
+    [Role.admins + Role.moderators].flatten.map {|role| super(type, id, role) }
+  end
+
+  def new_report(type, id, role)
     resource = {
-      :url => report_index_url,
-      :type => I18n.t('notifier.report_email.type.' + type),
-      :id => id
+      url:  report_index_url,
+      type: I18n.t("notifier.report_email.type." + type),
+      id:   id
     }
-
-    Role.admins.each do |role|
-      person = Person.find(role.person_id)
-      if person.local?
-        user = User.find_by_id(person.owner_id)
-        unless user.user_preferences.exists?(:email_type => :someone_reported)
-          resource[:email] = user.email
-          format(resource)
-        end
+    person = Person.find(role.person_id)
+    if person.local?
+      user = User.find_by_id(person.owner_id)
+      unless user.user_preferences.exists?(email_type: :someone_reported)
```

Comment:
Slight LoD violation: http://devblog.avdi.org/2011/07/05/demeter-its-not-just-a-good-idea-its-the-law/